include/handlers.h

```diff
@@ -298,20 +299,23 @@ class ImpiTask : public HssCacheTask
            int _impu_cache_ttl = 0,
            std::string _scheme_unknown = "Unknown",
            std::string _scheme_digest = "SIP Digest",
-           std::string _scheme_aka = "Digest-AKAv1-MD5",
```

Comment:
Yes - I thought this would be difficult to fix up, but on reflection I can probably make it be set in only two places, and maybe just in one.